.github/workflows/ci.yml

```diff
@@ -83,6 +74,15 @@ jobs:
       - name: Add OpenCL support - Windows
         if: ${{ matrix.os == 'windows-latest' }}
         run: choco install opencl-intel-cpu-runtime
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: set up go
+        uses: actions/setup-go@v4
+        with:
+          check-latest: true
+          go-version-file: "go.mod"
+      - name: setup env
+        run: make install
```

Comment:
what is the effect of changing the orders? these are also not described in the PR description.

Reply:
the idea is to prepare the VM/runner before checking out the code. Especially on windows this makes a big difference, first disabling the Windows defender and then checking out code is faster than the other way round (because Windows defender slows down the checkout).

Added to PR description.